lib/rules/length-zero-no-unit/__tests__/index.mjs

```diff
@@ -990,3 +990,30 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
```

Comment:
```suggestion
	ruleName,
	customSyntax: 'postcss-scss',
```

Although not needed, we typically signal non-standard tests with a custom syntax config.